odoo/addons/base/models/res_partner.py

```diff
@@ -556,11 +578,53 @@ def create(self, vals_list):
                 vals['image'] = self._get_default_image(vals.get('type'), vals.get('is_company'), vals.get('parent_id'))
             tools.image_resize_images(vals, sizes={'image': (1024, None)})
         partners = super(Partner, self).create(vals_list)
+
+        if self.env.context.get('skip_post_processing'):
+            return partners
+
         for partner, vals in pycompat.izip(partners, vals_list):
             partner._fields_sync(vals)
             partner._handle_first_contact_creation()
         return partners
 
+    def _load_records_create(self, vals_list):
+        partners = super(Partner, self.with_context(skip_post_processing=True))._load_records_create(vals_list)
```

Comment:
We should probably use a namespaced context key, e.g. `partner_field_sync` or something?

Reply:
renamed key to something namespaced & more specific.